services/src/datasets/external/netcdfcf/mod.rs

```diff
@@ -2695,7 +2695,7 @@ mod tests {
 
         assert_eq!(
             provider.load_layer(&layer_id).await.unwrap().metadata["dataRange"],
-            "[2.0,25093.0]"
+            "[1.0,98.0]"
```

Comment:
why does this change if the data doesn't change?

Reply:
Was incorrect previously, checked it with Panoply.